website/src/contribute/index.md

```diff
@@ -127,18 +131,24 @@ To contribute code, you need
        $ ./gradlew -p sdks/go check
        $ ./gradlew -p sdks/java/io/cassandra check
        $ ./gradlew -p runners/flink check
-
-1. Depending on the area you are updating, review tips for
+       
+1. Now you may want to set up your preferred IDE and other aspects of your development
+   environment. See the Developers' wiki for tips, guides, and FAQs on:
+   - [IntelliJ](https://cwiki.apache.org/confluence/display/BEAM/Using+IntelliJ+IDE)
+   - [Eclipse](https://cwiki.apache.org/confluence/display/BEAM/Eclipse+Tips)
```

Comment:
We agreed to get rid of Eclipse since it isn't really working/useful.

Reply:
Removed the link. I was trying not to discriminate.